test/nel-spec.js

```diff
@@ -107,6 +107,13 @@ describe('A Non-Empty immutable list', function () {
 
     })
 
+    it('will be filtered with filterNot', function () {
+        var n1 = NEL.fromList(List.fromArray([1, 2, 3, 4, 5, 6, 7, 8])).some()
+        expect(n1.filterNot(function (a) {
+            return a % 2 == 0
```

Comment:
`==` -> `===` (also mabe fix this in `filter` tests)

Reply:
This happens in many places. I can change it but I'd rather do it in a separate pull request for clarity.